network/iptables.md

```diff
@@ -58,4 +106,5 @@ for most users, because the `iptables` policies then need to be managed by hand.
 
 ## Next steps
 
-- Read [Docker Reference Architecture: Designing Scalable, Portable Docker Container Networks](https://success.docker.com/Architecture/Docker_Reference_Architecture%3A_Designing_Scalable%2C_Portable_Docker_Container_Networks)
+- Read [Docker Reference Architecture: Designing Scalable, Portable Docker Container Networks]
+(https://success.docker.com/Architecture/Docker_Reference_Architecture%3A_Designing_Scalable%2C_Portable_Docker_Container_Networks)
```

Comment:
looks like the added whitespace broke the link

To wrap links, the wrapping needs to be done in the links' "caption";

```Markdown
- Read [Docker Reference Architecture: Designing Scalable, Portable Docker
  Container Networks](https://success.docker.com/Architecture/Docker_Reference_Architecture%3A_Designing_Scalable%2C_Portable_Docker_Container_Networks)
```

That's a bit awkward sometimes, so an alternative could be to rephrase it slightly;

```Markdown
- Read "Docker Reference Architecture: Designing Scalable, Portable Docker
  Container Networks" on [success.docker.com](https://success.docker.com/Architecture/Docker_Reference_Architecture%3A_Designing_Scalable%2C_Portable_Docker_Container_Networks)
```

Or just don't wrap it (which I think in this case would be ok as an exception)